src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -18,17 +18,45 @@
 import com.google.appengine.api.users.UserServiceFactory;
 import com.google.gson.Gson;
 import com.google.sps.classes.SqlConstants;
+import com.google.sps.classes.Utility;
 import java.sql.*;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import javax.servlet.http.HttpServletRequest;
+import javax.sql.DataSource;
 
 // TODO(aabundis): Add JUnit tests for utility functions.
 
 /**
  * Utility methods used across classes. Just import class to access all methods.
  */
 public final class Utility {
-  // TODO(oumontiel): Move constants to different file.
+  // Define if running locally or deploying the current branch.
+  // Define LOCAL_OR_DEPLOYED constant as "local" for a local deployment or "deploy" for a cloud deployment.
+  public static final String LOCAL_OR_DEPLOYED = "local";
```

Comment:
Consider setting this as a boolean - if there are only two options, it's preferable to do it this way to guard against potential bugs. Could be something like 'IS_LOCAL_DEMO' or 'IS_LOCALLY_DEPLOYED'. Unless you think there might be more than two possible states for this!

Reply:
Ack!